scripts/seed-database/write/team_themes.sql

```diff
@@ -1,3 +1,5 @@
+-- TODO: Update to add action_color and link_colour columns
```

Comment:
Intentional `TODO`, will need to come back to pick this up post-deploy to production.